planner/core/common_plans.go

```diff
@@ -265,7 +265,9 @@ func (e *Execute) OptimizePreparedPlan(ctx context.Context, sctx sessionctx.Cont
 		preparedObj.Executor = nil
 		// If the schema version has changed we need to preprocess it again,
 		// if this time it failed, the real reason for the error is schema changed.
-		err := Preprocess(sctx, prepared.Stmt, is, InPrepare)
+		// FIXME: compatible with prepare
```

Comment:
The fix for `prepare` will be given soon?

Reply:
Or ban if it can not be done very soon.

Reply:
I create a issue for this: https://github.com/pingcap/tidb/issues/24932